app/models/booking.rb

```diff
@@ -2,10 +2,8 @@ class Booking < ApplicationRecord
   belongs_to :dress
   belongs_to :user
 
-  validates :dress, presence: true
-  validates :user, presence: true
-  validates :starting_date, presence: true
-  validates :ending_date, presence: true
+  validates :dress, :dress, :starting_date, :ending_date, :address, :post_code, :total_price, presence: true
```

Comment:
do we need the two :dress ? @Gazwai

Reply:
Updated the validates to only one ":dress". Nice catch man